content/topics/university-rdm-resources.md

```diff
@@ -147,22 +149,22 @@ The role of the data protection officer (dataskyddsombud) is to check that the G
     <tr>
       <td>Umeå University</td>
       <td><a href = "mailto:pulo@umu.se">pulo@umu.se</a></td>
-      <td><a href="https://www.aurora.umu.se/stod-och-service/rad-och-riktlinjer/juridik-och-personuppgifter/personuppgifter">Personuppgifter</a></td>
+      <td><a href="https://www.umu.se/en/about-the-website/legal-information/processing-of-personal-data/">Personuppgifter</a></td>
     </tr>
     <tr>
       <td>University of Gothenburg</td>
       <td><a href = "mailto:dataskydd@gu.se">dataskydd@gu.se</a></td>
```

Comment:
There is another email address: dataskyddsombud@gu.se. The email that we provide is for the dataskydd grupp but it works in some way. They offer these two alternatives as email contacts.